packages/orchestration/src/orchestration-types.ts

```diff
@@ -437,30 +438,36 @@ interface TranslationConfigParametersInput {
 }
 
 /**
- * Output parameters for translation configuration.
+ * Input parameters for translation input configuration.
  */
-interface TranslationConfigParametersOutput {
+export type TranslationInputParameters = TranslationConfigParametersInput & {
   /**
-   * Language of the text to be translated.
-   * @example sourceLanguage: 'de-DE'
+   * If true, the messages history will be translated as well.
+   * @example translateMessagesHistory: true
    */
-  sourceLanguage?: string;
+  translateMessagesHistory?: boolean;
   /**
-   * Language to which the text should be translated.
-   * @example targetLanguage: 'en-US'
+   * Configuration for applying translation to specific placeholders or message roles.
+   * @example applyTo: [{ category: 'placeholders', items: ['user_input'], source_language: 'de-DE' }]
    */
-  targetLanguage: string;
-}
+  applyTo?: SAPDocumentTranslationApplyToSelector[];
```

Comment:
[req] These fields can be added to `TranslationConfigParametersInput` instead of adding it as intersection type. I don't see an advantage of the intersection here.